src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.cs

```diff
@@ -39,6 +39,13 @@ public void TestAnalysisServicesServerScaleUpDown()
         }
 
         [Fact]
+        [Trait(Category.AcceptanceType, Category.LiveOnly)]
```

Comment:
The test contains SAS keys, which we do not want to expose them in source code. 
To run tests locally we set keys in enivronment variables, if there are any ways to get the SAS keys, please let me know.